src/resolve/resolved-bus.c

```diff
@@ -1292,6 +1299,37 @@ static int bus_property_get_dnssec_statistics(
                                      (uint64_t) m->n_dnssec_indeterminate);
 }
 
+static int bus_property_get_dnssec_supported(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        Manager *m = userdata;
+        DnsServer *server;
+        bool supported = true;
+        Iterator i;
+        Link *l;
+
+        assert(reply);
+        assert(m);
+
+        server = manager_get_dns_server(m);
+        if (server)
+                supported = supported && dns_server_dnssec_supported(server);
```

Comment:
hypothetically if !server, and !m->links, we'd still return supported=true, is that intentional?